docs/plugins/cloud-gce.asciidoc

```diff
@@ -74,38 +74,36 @@ If you did not set it yet, you can define your default project you will work on:
 gcloud config set project es-cloud
 --------------------------------------------------
 
-[[cloud-gce-usage-discovery-long-first-instance]]
-===== Creating your first instance
+[[cloud-gce-usage-discovery-long-login]]
 
+If you haven't already, login to Google Cloud
 
 [source,sh]
 --------------------------------------------------
-gcutil addinstance myesnode1 \
-       --service_account_scope=compute-rw,storage-full \
-       --persistent_boot_disk
+gcloud auth login
 --------------------------------------------------
 
-Then follow these steps:
+This will open your browser. You will be asked to sign-in to a Google account and
+authorize access to the Google Cloud SDK.
 
-* You will be asked to open a link in your browser. Login and allow access to listed services.
-* You will get back a verification code. Copy and paste it in your terminal.
-* You should see an `Authentication successful.` message.
-* Choose your zone, e.g. `europe-west1-a`.
-* Choose your compute instance size, e.g. `f1-micro`.
-* Choose your OS, e.g. `projects/debian-cloud/global/images/debian-7-wheezy-v20140606`.
-* You may be asked to create a ssh key. Follow the instructions to create one.
+[[cloud-gce-usage-discovery-long-first-instance]]
+===== Creating your first instance
+
+
+[source,sh]
+--------------------------------------------------
+gcloud compute instances create myesnode1 \
+       --zone <your-zone> \
+       --scopes compute-rw
+--------------------------------------------------
```

Comment:
I believe `storage-full` might be required. I got this error without it:

```
$ gsutil mb gs://esimage

Updates are available for some Cloud SDK components.  To install
them, please run:
 $ gcloud components update

Creating gs://esimage/...
AccessDeniedException: 403 Insufficient Permission
```

I am trying to spin up a new instance now _with_ `storage-full`. If the error goes away, I'll update the PR.